### PR TITLE
Run Babel and Prettify on tweenruntimebehavior.js for backwards compatibility

### PR DESCRIPTION
**Pull request by Wend1go**
Hope the history is cleaner now

### CODE REVIEW

**Comment by 4ian** [comment]
Cool, I'll take a look when back at home. If think that comments were moved though out of their proper lines :'(

File: Extensions/TweenBehavior/tweenruntimebehavior.js [hunk 1]
```diff
@@ -418,7 +426,6 @@ gdjs.TweenRuntimeBehavior.prototype.addObjectPositionYTween = function(
 
   this._setupTweenEnding(identifier, destroyObjectWhenFinished);
 };
-
```
Comment (4ian) [inline Extensions/TweenBehavior/tweenruntimebehavior.js]: There are "random" changes like these which seems to move comments out of their original functions

File: Extensions/TweenBehavior/tweenruntimebehavior.js [hunk 2]
```diff
@@ -511,9 +521,10 @@ gdjs.TweenRuntimeBehavior.prototype.addObjectScaleTween = function(
     },
     duration: durationValue,
     easing: easingValue,
-    step: state => {
-      this.owner.setScaleX(state.scaleX);
-      this.owner.setScaleY(state.scaleY);
+    step: function step(state) {
+      _this6.owner.setScaleX(state.scaleX);
+
```
Comment (4ian) [inline Extensions/TweenBehavior/tweenruntimebehavior.js]: Same here, not a big deal but an empty line was added

File: Extensions/TweenBehavior/tweenruntimebehavior.js [hunk 3]
```diff
@@ -949,75 +969,67 @@ gdjs.TweenRuntimeBehavior.makeNewTweenable = function(runtimeScene) {
...
  * @private
  */
+
 gdjs.TweenRuntimeBehavior.gdjsCallbackRuntimeSceneUnloaded = function(
   runtimeScene
 ) {
-  if (!runtimeScene.shiftyJsScene) return;
-
-  // Stop and explictly remove all tweenables to be sure to drop
+  if (!runtimeScene.shiftyJsScene) return; // Stop and explictly remove all tweenables to be sure to drop
```
Comment (4ian) [inline Extensions/TweenBehavior/tweenruntimebehavior.js]: This change is quite bad because it moved the comment at a place that is not related to the comment, making it misleading

File: Extensions/TweenBehavior/tweenruntimebehavior.js [hunk 4]
```diff
@@ -949,75 +969,67 @@ gdjs.TweenRuntimeBehavior.makeNewTweenable = function(runtimeScene) {
...
     .getTimeFromStart();
-  runtimeScene.shiftyJsScene.resume();
-
-  // Note that per the invariant of shiftyJsScene, shiftyJsScene will only
+  runtimeScene.shiftyJsScene.resume(); // Note that per the invariant of shiftyJsScene, shiftyJsScene will only
   // contains tweenables that should be playing (so calling resume is safe).
-};
-
-// Handle Shifty.js updates (the time and the "tick" of tweens
+}; // Handle Shifty.js updates (the time and the "tick" of tweens
```
Comment (4ian) [inline Extensions/TweenBehavior/tweenruntimebehavior.js]: Same issue here

**Comment by 4ian** [comment]
Unfortunately Babel messed up with the comments in the code (which could be normally ok if it was just used as a tool to transform JS). In our case, this is not acceptable because it's messing up the source code, even making comments misleading because they are not on their proper lines anymore.

So it's a pity to ask this but could you instead apply the fixes manually?
* `() =>` should be `function`. Add if necessary a `var that = this;` Use `that` as the name for the variable, as it's an idiom used in the rest of the codebase.
* `const` should be replaced by `var`.

Let me know if that's ok for you! :)

**Comment by Wend1go** [comment]
Finally got everything up an running again after installing Linux Mint 19.1.
I'll do the manual changes at the weekend.

**Comment by Wend1go** [comment]
Ok finished

**Comment by 4ian** [comment]
Looks good, merged! Thanks for doing this :)
